cf-nodejs-logging-support-core/log-core.js

```diff
@@ -292,9 +294,15 @@ var initLog = function () {
     var logObject = JSON.parse(initDummy);
 
     var now = new Date();
-
     logObject.written_at = now.toJSON();
-    logObject.written_ts = now.getTime() * NS_PER_MS + (process.hrtime()[1] % 1000000);
+
+    var lower = process.hrtime()[1] % 1000000
```

Comment:
Can you use `NS_PER_MS` instead of `1000000`?

Reply:
Thats a good catch